pkg/core/runtime/runtime.go

```diff
@@ -145,6 +151,8 @@ type runtimeContext struct {
 	metrics        metrics.Metrics
 	erf            events.ListenerFactory
 	apim           api_server.APIInstaller
+	xdsauth        xds_auth.Authenticator
+	xdsCallbacks   util_xds.Callbacks
```

Comment:
This always feels like more code than there should be to plug a runtime config :)